changelog.d/3510.feature

```diff
@@ -0,0 +1 @@
+Expose monthly active user data via _matrix/client/api/v1/admin/server_metrics
```

Comment:
r0 not v1, surely? (also, should the admin endpoints be specced?)

Reply:
ah yes, good spot